packages/shared-ux/button_toolbar/src/buttons/toolbar_button/toolbar_button.styles.ts

```diff
@@ -24,9 +24,9 @@ export const ToolbarButtonStyles = ({ euiTheme }: UseEuiTheme) => {
       borderColor: euiTheme.border.color,
     },
     emptyButton: {
-      backgroundColor: euiTheme.colors.emptyShade,
+      backgroundColor: euiTheme.colors.backgroundBasePlain,
       border: `${euiTheme.border.thin}`,
-      color: `${euiTheme.colors.text}`,
+      color: `${euiTheme.colors.textParagraph}`,
```

Comment:
Accessing this from Kibana:
https://github.com/user-attachments/assets/ecac7233-aa31-4724-b27c-57ef9219be40